src/status_im2/contexts/communities/menus/channel_options/view.cljs

```diff
@@ -0,0 +1,116 @@
+(ns status-im2.contexts.communities.menus.channel-options.view
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [quo2.core :as quo]
+            [status-im2.common.mute-chat-drawer.view :as mute-chat-drawer]
+            [utils.datetime :as datetime]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:hide-bottom-sheet])
+  (rf/dispatch event))
+
+(defn mute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:show-bottom-sheet
+                            {:content (fn []
+                                        [mute-chat-drawer/mute-chat-drawer chat-id
+                                         :mute-chat-for-duration])}]))
+
+(defn unmute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/mute chat-id false 0]))
+
+(defn view-members
+  [_id]
+  {:icon                :i/members
+   :accessibility-label :view-members
+   :label               (i18n/label :t/view-channel-members-and-details)})
+
+(defn view-token-gating
+  [id]
+  {:icon                :i/token
+   :right-icon          :i/chevron-right
+   :chevron?            true
+   :accessibility-label :view-token-gating
+   :on-press            #(js/alert (str "implement action" id))
+   :label               (i18n/label :t/view-token-gating)})
+
+(defn mark-as-read
+  [_id]
+  {:icon                :i/up-to-date
+   :accessibility-label :mark-as-read
+   :label               (i18n/label :t/mark-as-read)
+   :on-press            #()})
```

Comment:
Maybe just leave it without `on-press`. It's obvious from the surrounding code these are option actions.